docs/cita/sys-contract-interface/interface.md

```diff
@@ -1801,6 +1801,26 @@ $ scm SysConfig setWebsite \
         --admin-private 0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6 \
 ```
 
+### setBlockInterval
+
+动态设置出块时间。
+
+* 参数
+
+    `Integer` - 出块时间(毫秒)
+
+* 返回值
+
+    空
+
+* 示例
+
+```shell
+$ scm SysConfig setBlockInterval \
```

Comment:
For a runnable shell command, It should be:

```shell
cita-cli scm ...
```
Yeah, as you magic, we need to change some commands in this file.

Reply:
Yes！ That’s a big problem. Another pr for this flaw is nessary.

Reply:
Okay.